common/kafka/stats_enum.h

```diff
@@ -24,6 +24,7 @@
 enum CounterIdx {
```

Comment:
I am not sure how this would be used?

For our stats library, it only allows to register one metric vector and one counter vector in one process. If a main service already has the stats vectors, we will have to use the "string" type stat name in libraries like common/kafka here.

Reply:
Done